packages/text-sets/package.json

```diff
@@ -42,7 +42,7 @@
     "@googleforcreators/migration": "*"
   },
   "devDependencies": {
-    "puppeteer": "*"
```

Comment:
This can actually stay `*`. The version in the main `package.json` is what matters.